src/app/components/modules/Modals.jsx

```diff
@@ -82,11 +90,16 @@ class Modals extends React.Component {
                 'https://blocktrades.us/?input_coin_type=eth&output_coin_type=steem_power&receive_address=' +
                 username;
         };
-
         return (
             <div>
                 {show_login_modal && (
-                    <Reveal onHide={hideLogin} show={show_login_modal}>
+                    <Reveal
+                        onHide={() => {
+                            hideLogin();
+                            loginBroadcastOperation.errorCallback();
```

Comment:
Just to be clear - this will automatically trigger errorCallback any time the login modal is shown and cancelled? Two concerns: (1) do any callers already handle this on their own, (2) is onHide only for cancel or is it possible to be triggered when the dialog closes because of success?

Reply:
Yes very good points, I'm going to undo this and simply not display a subscribe action pending on the subscribe button when an unauthenticated user clicks it - because a subscribe action isn't pending until they are authenticated.